src/test/java/steps/ApplicationSteps.java

```diff
@@ -0,0 +1,24 @@
+package steps;
+
+import cucumber.api.java8.En;
+
```

Comment:
fine for now, but we need to replace these with some useful pre / post conditions (i.e. starting up our app) and steps (i.e. calling endpoints)